src/components/ModalVue.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="modal__container">
+    <div class="modal__overlay"></div>
+    <div class="modal__main">
+      <p>Are You Sure?</p>
+
+      <div class="modal__main_button">
+        <button class="modal__main_button-primary btn" @click="$emit('yes')">
+          Yes
+        </button>
+        <button class="modal__main_button-secondary btn" @click="$emit('no')">
+          No
+        </button>
```

Comment:
```suggestion
        <button class="modal__main_button-primary btn" @click="$emit('confirm')">
          Yes
        </button>
        <button class="modal__main_button-secondary btn" @click="$emit('cancel')">
          No
        </button>
```

Also usually we should use methods for `@click`, instead of directly using $emit. The reason we do this is to make the HTML more readable as HTML can be really messy if not taken care properly.